source/adapters/level_zero/command_buffer.cpp

```diff
@@ -27,31 +27,6 @@ template <typename T> T *getPointerFromVector(std::vector<T> &V) {
   return V.size() == 0 ? nullptr : V.data();
 }
 
-/**
- * Checks the version of the level-zero driver.
- * @param[in] Context Execution context
- * @param[in] VersionMajor Major version number to compare to.
- * @param[in] VersionMinor Minor version number to compare to.
- * @param[in] VersionBuild Build version number to compare to.
- * @return true if the version of the driver is higher than or equal to the
- * compared version.
- */
-bool isDriverVersionNewerOrSimilar(ur_context_handle_t Context,
```

Comment:
Think the callsite in command_buffer.cpp will need updated since we changed the first character to uppercase `I`

Reply:
Thanks, will update. Just noticed that failure now after rebasing around the conflicts.